engine/security/certificates.md

```diff
@@ -29,8 +29,10 @@ A custom certificate is configured by creating a directory under
 `localhost`). All `*.crt` files are added to this directory as CA roots.
 
 > **Note:**
-> In the absence of any root certificate authorities, Docker
-> will use the system default (i.e., host's root CA set).
+> As of docker 1.13, on Linux any root certificates authorities will be merged
+> in with the system defaults (i.e., host's root CA set). Prior to 1.13 and on
+> Windows, the system default certificates will only be used when there are no
```

Comment:
@joaofnfernandes restart is not needed, these certificates are always read at the start of a push or pull operation and not kept around after